mountpoint-s3-crt/src/common/uri.rs

```diff
@@ -64,6 +64,16 @@ impl Uri {
         }
     }
 
+    /// Return the host port portion of the URI. If no port was present, returns 0
+    pub fn host_port(&self) -> u16 {
+        // SAFETY: `inner` is a valid `aws_uri` since it's owned by this struct, and the lifetime of
+        // the returned slice will be tied to &self.
+        unsafe {
+            let cursor = aws_uri_port(self.to_inner_ptr());
+            cursor
```

Comment:
No need to assign to new variable.
```suggestion
            aws_uri_port(self.to_inner_ptr())
```

Reply:
Ack